src/index.js

```diff
@@ -58,6 +58,9 @@ import handleStartupEvent from './squirrel';
     mainWindow = new BrowserWindow(generateBrowserConfig());
     global.mainWindowID = WindowManager.add(mainWindow, 'main');
 
+    // Required for 'darwin'
+    mainWindow.setFullScreenable(true);
```

Comment:
Hm, strange.  Electron docs are not the most precise things in the world.

If it works it works :+1: